environment/README.md

```diff
@@ -1,77 +1,132 @@
-# Manual conda environment installation and data download
+# Python environment and data download
 
-If you want to run the scripts/notebooks, you need to follow these steps to create a conda environment and download the necessary data.
+If you want to run the scripts/notebooks, you need to follow these steps to create a Python environment and download the necessary data.
+Although you can create your own Python environment, using our Docker image is much easier.
 
 Keep in mind that although unit tests are automatically run on Linux, macOS and MS Windows, the software is manually tested only on Linux/Ubuntu.
 
-1. Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html) or Anaconda.
+There are four steps:
+1. Getting the source code (cloning this repository).
+1. Adjusting settings (paths, etc). 
+1. Creating a Python environment.
+1. Downloading the data to run the analyses.
 
-2. Open a terminal, clone this repository, and `cd` into the repository root folder.
+## Getting the source code
 
-3. Run `cd environment`.
+```bash
+git clone https://github.com/greenelab/ccc.git
+cd ccc/
+```
 
-4. (optional) Adjust your environment variables:
+## Adjusting settings
 
-    ```bash
-    # (optional, will default to subfolder 'cm_gene_expr' under the system's temporary directory)
-    # Root directory where all data will be downloaded to
-    export CM_ROOT_DIR=/tmp/cm_gene_expr
+Adjust the paths where the data will be downloaded, the number of CPU cores available to run the code, etc.
+You can specify these options using environment variables:
```

Comment:
Do you mean to say that you can change these variables in `settings.py`?

Reply:
I made some changes to hopefully make this clear.